src/components/Calendar.scss

```diff
@@ -0,0 +1,162 @@
+.Calendar-heading, .Calendar-secondary-heading {
+  height: 60px;
+}
+.Calendar-title, .Calendar-month {
+  margin-right: $su-medium;
+  display: inline-block;
+}
+.Calendar-secondary-heading {
+  padding: $su-medium;
+  border-left: 1px solid $ts-grey;
+  border-right: 1px solid $ts-grey;
+  border-top: 1px solid $ts-grey;
+}
+.Calendar-row[data-ui-component="week-heading"] li {
+  display: none;
+}
+.Calendar-row[data-ui-component="week-heading"] {
+  background: $cu-background--light;
+  margin: 0;
+  padding: $su-small 0;
+  border: 1px solid $ts-grey;
+  color: $cu-primary;
+  list-style-type: none;
+}
+.Calendar-row[data-ui-component="week"] {
+  border-right: 1px solid $ts-grey;
+  & > [class*="Calendar-cell"] {
+    height: auto;
+    padding-bottom: $su-small;
+    border-left: 1px solid $ts-grey;
+    border-bottom: 1px solid $ts-grey;
+    span {
```

Comment:
Can this be written as a block level element instead, like a `<p>`?

Reply:
Sure